webex-calling-calls-api.json

```diff
@@ -1,8 +1,8 @@
 {
 	"info": {
-		"_postman_id": "cda01864-ad4d-4de3-b243-f54afa27b62a",
-		"name": "Webex Calling Calls API",
-		"description": "These tests exercise the /telephonycalls Webex Calling API and validate the results.  They are designed to provide developers who are unfamilar with the Calling API with a set of examples to understand the basic concepts of how to use them.\n\n1) Create an outbound (Click to Dial) Call and verify it is gettable and perform a basic Hold and Resume operation on the call.\n2) Answer an inbound call (UserA calls apiUser), intiatate another call from the same user (apiUser calls UserB) and perform a Consult Transfer (i.e. apiUser transfers the call to connect UserA with UserB)\n3) Answer an inbound call (UserA calls apiUser) and blind transfer (Divert) the call to apiUser's voicemail.\n4) Answer an inbound call, park the call on another destination and retrieve the call back to apiUser.\n5) Get Call History (i.e. missed calls, placed calls and received calls).\n6) Voice Messages (execute the voice messages APIs)\n\nBefore running these tests the following environment variables must be set:\n* WEBEX_TOKEN -- an OAuth token for a user configured for calling in the test environment. These tests require a token with all the calling scopes (spark:calls_read and spark:calls_write).  To get started quickly, developers can copy their temporary token from the [Webex For Developers Gettings Started Guide](https://developer.webex.com/docs/api/getting-started#accounts-and-authentication). \n* WEBEX_API_URL -- the URL of the meetings API under test, generally the default value of \"https://webexapis.com/v1/\" does not need to be changed\n* DESTINATION -- The destination to be dialed. Destination can be extensions, PSTN numbers, FAC codes or SIP URI. The destination can be digits or a URI. Some examples for destination include: 1234, 2223334444, +12223334444, *73, tel:+12223334444, user@company.domain, sip:user@company.domain\n* TRANSFER_DESTINATION -- The destination to be dialed. This is used to make a second call from the api user in order to transfer the call.\n* PARK_DESTINATION  -- The destination to park the call.\n* VOICE_MESSAGE_ID - the voice message Id.  Used by the Voice Message APIs that require a message Id to perform the action.",
+		"_postman_id": "e0ebe617-709b-41c3-9785-ee5e5aa49d0c",
+		"name": "Webex Calling Calls API Copy",
```

Comment:
Can we leave the name as Webex Calling Calls API

Reply:
My mistake, I forgot to change/rename it after I imported it into Postman - with that name already in-use.
It has been changed back to the original name.